tests/scripts/dind-cluster.sh

```diff
@@ -180,18 +183,20 @@ function dind::check-image {
 
 function dind::filter-make-output {
   # these messages make output too long and make Travis CI choke
-  egrep -v --line-buffered 'I[0-9][0-9][0-9][0-9] .*(parse|conversion|defaulter|deepcopy)\.go:[0-9]+\]'
+   grep -E -v --line-buffered 'I[0-9][0-9][0-9][0-9] .*(parse|conversion|defaulter|deepcopy)\.go:[0-9]+\]'
```

Comment:
Accidental extra space?
```suggestion
  grep -E -v --line-buffered 'I[0-9][0-9][0-9][0-9] .*(parse|conversion|defaulter|deepcopy)\.go:[0-9]+\]'
```